runtime/mainnet/src/lib.rs

```diff
@@ -98,9 +104,33 @@ pub type TxExtension = (
 	CheckMetadataHash<Runtime>,
 );
 
+/// EthExtra converts an unsigned Call::eth_transact into a CheckedExtrinsic.
+#[derive(Clone, PartialEq, Eq, Debug)]
+pub struct EthExtraImpl;
+
+impl pallet_revive::evm::runtime::EthExtra for EthExtraImpl {
+	type Config = Runtime;
+	type Extension = TxExtension;
+
+	fn get_eth_extension(nonce: u32, tip: Balance) -> Self::Extension {
+		(
+			CheckNonZeroSender::<Runtime>::new(),
+			CheckSpecVersion::<Runtime>::new(),
+			CheckTxVersion::<Runtime>::new(),
+			CheckGenesis::<Runtime>::new(),
+			CheckMortality::from(generic::Era::Immortal),
```

Comment:
This one is interesting.
I guess on EVM land transactions don't become invalid after a certain period of time. If I remember correctly one can always replace an already sent tx if this one hasn't been included yet by using the same nonce and assigning to it a higher gas.

Reply:
That is intersting, thanks for pointing out